File: CA2_MobileProject/CA2_MobileProject/Models/Event.cs
namespace CA2_MobileProject.Models
{
    public class Event
    {
        public int EventId { get; set; }
        public string Title { get; set; }
        public DateTime DateTime { get; set; }
        public string Location { get; set; }
        public string Category { get; set; }
        public string? Description { get; set; }
        public decimal? Price { get; set; }
    }
}

File: EventFinder/app/src/main/java/com/ca2/eventfinder/model/Event.kt
package com.ca2.eventfinder.model

data class Event(
    val eventId: Int = 0,
    val title: String,
    val dateTime: String,
    val location: String,
    val category: String,
    val description: String,
    val price: Double,
    val createdByUserId: Int
)